vsintegration/src/FSharp.VS.FSI/sessions.fs

```diff
@@ -154,25 +154,37 @@ let catchAll (ie: IEvent<_,_>) : IEvent<_> =
     ie.Add(fun x -> try w(x) with err -> ignore(System.Windows.Forms.MessageBox.Show(err.ToString())); ());
     e
 
-let fsiExeName () = if SessionsProperties.useAnyCpuVersion then "fsianycpu.exe" else "fsi.exe"
-let determineFsiRelativePath () =
+let fsiExeName () = 
+    if SessionsProperties.useAnyCpuVersion then "fsiAnyCPU.exe" else "fsi.exe"
+
+// Use the VS-extension-installed development path if available, relative to the location of this assembly
+let determineFsiRelativePath1 () =
```

Comment:
This addition uses the FSI.EXE installed by ``EnableOpenSource.vsix`` if it is available.